isis/tests/spiceinitTests.cpp

```diff
@@ -11,43 +9,17 @@
 #include "PvlGroup.h"
 #include "PvlKeyword.h"
 #include "TestUtilities.h"
+#include "FileName.h"
+
+#include "Fixtures.h"
 
 #include "gmock/gmock.h"
 
 using namespace Isis;
 
-class spiceinitTestCube : public ::testing::Test {
-  protected:
-    Cube testCube;
-    QTemporaryFile tempFile;
-
-  void SetUp() override {
-    tempFile.open();
-    testCube.setPixelType(UnsignedByte);
-  }
-
-  void TearDown() override {
-    if (testCube.isOpen()) {
-      testCube.close();
-    }
-  }
-
-  void createCube(Pvl &label) {
-    PvlObject cubeLabel = label.findObject("IsisCube");
-    PvlGroup dimensions = cubeLabel.findObject("Core").findGroup("Dimensions");
-    testCube.setDimensions(dimensions["Samples"],
-                          dimensions["Lines"],
-                          dimensions["Bands"]);
-    testCube.create(tempFile.fileName());
+QString APP_XML = FileName("$ISISROOT/bin/xml/spiceinit.xml").expanded();
 
-    for (auto grpIt = cubeLabel.beginGroup(); grpIt!= cubeLabel.endGroup(); grpIt++) {
-      testCube.putGroup(*grpIt);
-    }
-  }
-};
-
-
-TEST_F(spiceinitTestCube, PredictAndReconCK) {
+TEST_F(TestCube, spiceinit_predict_and_recon_ck) {
```

Comment:
Not a big deal, just wondering if we are swapping to using underscores instead of camel case for test names.

Reply:
This should be back to camel case I think.